slack_today_i_did/parser.py

```diff
@@ -72,6 +72,10 @@ def eval(tokens, known_functions, default_args=None):
         first_function_name = 'error-help'
         args.append(('NO_TOKENS', str))
     # when we have stuff to work with!
+    elif 'help' == tokens[0][1]:
```

Comment:
Tried something here: https://github.com/NoRedInk/slack-today-i-did/commit/5ca2dfc472f8ff29eac4421b3bdc8d5bd1264a7e
not sure about the indirection this approach causes